canonical-web-and-design/maas-ui
# Init legacy ui

## Done
* Created new `legacy` repo containing angularjs maas client.
* Updated all instances of `templateUrl` to use html loaded from the webpack `html-loader` and es6 imports.
* Removed all '3rd party' dependencies, all dependencies now from npm. Upgrade to AngularJS LTS.
* Setup webpack dev server with hot reloading.
* Reorganised/simplified directory structure.

## QA
1. copy `.env` to `.env.local` and point to your MAAS.
2. run `yarn`
3. run `yarn start`
4. In another tab, login to MAAS using the old maas login.
5. Copy the csrf token from your other tab into the current tag running the legacy client. You should see the angularjs client running!

## Also closes
https://github.com/canonical-web-and-design/MAAS-squad/issues/1600
https://github.com/canonical-web-and-design/MAAS-squad/issues/1599
https://github.com/canonical-web-and-design/MAAS-squad/issues/1601
https://github.com/canonical-web-and-design/MAAS-squad/issues/1603
